### PR TITLE
Samsung.TizenStudio version 5.0

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/98731)

### CODE REVIEW

File: manifests/s/Samsung/TizenStudio/5.0/Samsung.TizenStudio.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Samsung.TizenStudio
+PackageVersion: 5.0
+PackageLocale: en-US
+Publisher: Samsung
+PackageName: Tizen Studio
```
Comment (stephengillie) [inline manifests/s/Samsung/TizenStudio/5.0/Samsung.TizenStudio.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://developer.samsung.com/
PublisherSupportUrl: https://developer.samsung.com/support
PrivacyUrl:   https://www.tizen.org/about/privacy-policy
PackageName: Tizen Studio
PackageUrl: https://developer.tizen.org/development/tizen-studio/download
```

**Comment by stephengillie** [comment]
Hi @upintheairsheep,

The installation isn't unattended - the installer requires a keypress to exit:
image

Is there an installer switch to bypass this and have it exit automatically?

**Comment by upintheairsheep** [comment]
> Hi @upintheairsheep,
> 
> The installation isn't unattended - the installer requires a keypress to exit: image
> 
> Is there an installer switch to bypass this and have it exit automatically?

I'm not sure, try /s or /S

File: manifests/s/Samsung/TizenStudio/5.0/Samsung.TizenStudio.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+PackageIdentifier: Samsung.TizenStudio
+PackageVersion: 5.0
+Installers:
+- InstallerUrl: https://download.tizen.org/sdk/Installer/tizen-studio_5.0/web-cli_Tizen_Studio_5.0_windows-64.exe
+  Architecture: x64
+  InstallerType: exe
+  InstallerSha256: 73E767B1593D145339D5C269AEC4D7BEB94E10E41886996EA1409EE80FFA632C
+  InstallerSwitches:
+    Silent: -s
+    SilentWithProgress: /passive
```
Comment (stephengillie) [inline manifests/s/Samsung/TizenStudio/5.0/Samsung.TizenStudio.installer.yaml:13]:
```suggestion
```

**Comment by stephengillie** [comment]
This package doesn't appear to have a silent install option. Examining the suggested switches only gives 3 options, 2 related to the license and 1 related to Java:
```
Loading... Please wait.
Setting up the jdk in temp folder
Invalid options : -s

Usage: tizen-web-cli_XXX [option]... [<installation path>]
       tizen-web-cli_XXX [--show-license] [--accept-license] [--no-java-check] [
<installation path>]

Options:
  --show-license         display Tizen Studio software license agreement.
  --accept-license       accept license agreement.
...

```

Pending: 
* https://github.com/microsoft/winget-cli/issues/910

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Pull request closed without merging**